packages-ts/starknet-gauntlet/test/encoding/encoding.test.ts

```diff
@@ -12,8 +11,8 @@ export function bytesToFeltsDeprecated(data: Uint8Array): string[] {
   // chunk every 31 bytes
   for (let i = 0; i < data.length; i += CHUNK_SIZE) {
     const chunk = data.slice(i, i + CHUNK_SIZE)
-    // cast to int
-    felts.push(new BN(chunk, 'be').toString())
+    const feltHex = `0x${Array.from(chunk, (byte) => byte.toString(16).padStart(2, '0')).join('')}`
```

Comment:
Hmm, what's happening here?

Can we extract this in a fn with a name and a potential comment?

Reply:
So this line is converting 31 bytes to a decimal string. But instead of using bn.js, we are converting each byte to a string and concatenating all the strings together. Then we turn the resultant hexadecimal string to a bigint and then convert it to a decimal string. I'll extract this into a function

Reply:
The .padStart is to ensure strings such as 'f' are represented by '0f' (pad with 0s) so each byte is represented by two characters so that when you concatenate the string no places are omitted

Reply:
This is just a compat test with the old implementation, we could remove it altogether 👍🏻